hirola/front/static/front/css/about.css

```diff
@@ -1,6 +1,6 @@
 
```

Comment:
You can also remove these spaces it seems no change was done on this one. Just to clean it up